content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -73,15 +71,20 @@ $ curl \
 
 ## Add Team access to a Workspace
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /team-workspaces |
+`POST /team-workspaces`
 
-### Parameters
+### Request Body
 
-- `access` (`string: <required>`) - `read`, `write`, or `admin`
-- `team` (`string: <required>`) - The ID of the team to add to the workspace.
-- `workspace` (`string: <required>`) - The workspace ID to which the team is to be added.
+This POST endpoint requires a JSON object with the following properties as a request payload. (Properties without a default value are required.)
+
+Key path                                 | Type   | Default | Description
+-----------------------------------------|--------|---------|------------
+`data.type`                              | string |         | Must be `"team-workspaces"`.
```

Comment:
In the overview section for `JSON API` it might be worthwhile summarising these concepts. `type`, `attributes` and `relationships` are key for us and could use a sentence or two.

Reply:
Agree, but I want more clarity about what they're for before I can write that. TBH they look like busywork boilerplate to me, but I assume they give you some benefit on the backend.

Reply:
I think some one-liners could be pulled from [Resource Objects](http://jsonapi.org/format/#document-resource-objects), but unsure if they'd translate to our documentation cleanly.